crates/polars-ops/src/chunked_array/mod.rs

```diff
@@ -3,8 +3,6 @@ pub mod array;
 mod binary;
 #[cfg(feature = "timezones")]
 pub mod datetime;
-#[cfg(feature = "interpolate")]
-mod interpolate;
```

Comment:
I'm moving this out of `chunked_array` and into `series`, as the public function operates on (and returns) a Series